src/services/workbench/layoutService.ts

```diff
@@ -48,6 +53,16 @@ export interface ILayoutService extends Component<ILayout> {
      * @unachieved
      */
     setSideBarPosition(position: keyof typeof Position): void;
+    /**
+     * Set the mode of the menu bar, default is `vertical`
+     * @param mode
+     * @unachieved
+     */
+    setMenuBarMode(mode: keyof typeof MenuBarMode): void;
+    /**
+     * Get the mode of the menu bar
```

Comment:
```suggestion
     * Get the mode of the MenuBar
```

MenuBar 作为我们独立的组件命名，这样跟文档应该更好衔接